src/include/session.h

```diff
@@ -294,12 +294,13 @@ struct __wt_session_impl {
 #define WT_SESSION_LOGGING_INMEM 0x001000u
 #define WT_SESSION_NO_DATA_HANDLES 0x002000u
 #define WT_SESSION_NO_RECONCILE 0x004000u
-#define WT_SESSION_PREFETCH 0x008000u
-#define WT_SESSION_QUIET_CORRUPT_FILE 0x010000u
-#define WT_SESSION_READ_WONT_NEED 0x020000u
-#define WT_SESSION_RESOLVING_TXN 0x040000u
-#define WT_SESSION_ROLLBACK_TO_STABLE 0x080000u
-#define WT_SESSION_SCHEMA_TXN 0x100000u
+#define WT_SESSION_PREFETCH_ENABLED 0x008000u
+#define WT_SESSION_PREFETCH_THREAD 0x010000u
```

Comment:
Do comments play nicely with `flags.py`? If so, I think the fact that we have two flags for prefetch could use a little explanation. Either that or we move one of them to a `bool`, or perhaps an even larger change - e.g. perhaps it's enough for `__inmem_row_leaf` to check that a session is internal, rather than prefetch specifically.

Reply:
As discussed offline, we will leave the flag as is and add a comment explaining further context.